sensing/pointcloud_preprocessor/src/utility/utilities.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright 2022 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "pointcloud_preprocessor/utility/utilities.hpp"
+
+namespace pointcloud_preprocessor::utils
+{
+std::vector<PointCgal> to_cgal_polygon(
```

Comment:
It may be better to use it as a pointer or argument, since NRVO is not guaranteed(copying may occur).

Reply:
@yukkysaito 
modified to use argument instead of return value.

Reply:
LGTM